.github/workflows/on-chain-upgrade.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      GENESIS_DESMOS_VERSION: "v4.1.0"
-      GENESIS_URL: "https://raw.githubusercontent.com/RiccardoM/desmos-states/master/morpheus-apollo-2-6608740.json"
-      UPGRADE_NAME: "v4.3.0"
+      GENESIS_DESMOS_VERSION: "v4.3.0"
+      GENESIS_URL: "https://raw.githubusercontent.com/RiccardoM/desmos-states/master/morpheus-apollo-2-7199960.json"
```

Comment:
The current file is in git lfs now.
It should be downloaded in this [guide](https://gist.github.com/fkraeutli/66fa741d9a8c2a6a238a01d17ed0edc5#retrieving-lfs-files)

Reply:
I fixed the issue by branch https://github.com/desmos-labs/desmos/tree/paul/fix-release-v4.4.1

Reply:
@dadamu Can you open a PR from your branch to this one? This way we can run the CI on it and see if that fixes the on-chain upgrade test

Reply:
Opened #1003